controllers/fenceagentsremediation_controller.go

```diff
@@ -79,7 +80,16 @@ func (r *FenceAgentsRemediationReconciler) Reconcile(ctx context.Context, req ct
 		r.Log.Error(err, "failed to get FAR CR")
 		return emptyResult, err
 	}
-	// TODO: Validate FAR CR name to nodeName. Run isNodeNameValid
+	// Validate FAR CR name to match a nodeName from the cluster
+	r.Log.Info("Check FAR CR's name")
+	valid, err := farUtils.IsNodeNameValid(r.Client, req.Name)
```

Comment:
Nit: I  prefer `utils` over `farUtils`.
in any case this is the name of the pkg and it's clear we are in the context of far.